packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -13,15 +18,25 @@ interface Props {
   hideHorizontalTrack?: boolean;
   hideVerticalTrack?: boolean;
   scrollTop?: number;
-  setScrollTop: (event: any) => void;
+  setScrollTop?: (event: any) => void;
   autoHeightMin?: number | string;
   updateAfterMountMs?: number;
+  isPageScrollbar?: boolean;
+}
+
+interface StylesInterface {
+  customScrollbar: string;
+  trackVertical: string;
+  trackHorizontal: string;
+  thumbVertical: string;
+  thumbHorizontal: string;
+  view: string;
```

Comment:
nit: Maybe move this closer to the `getStyles` function so you can easily see the component props.